jwst/nsclean/nsclean.py

```diff
@@ -134,8 +133,11 @@ def create_mask(input_model, mask_spectral_regions, n_sigma):
     # Initialize mask to all True. Subsequent operations will mask
     # out pixels that contain signal.
     # Note: mask will be 3D for BOTS mode data
-    mask = np.full(np.shape(input_model.dq), True)
+    #mask = np.full(np.shape(input_model.dq), True)
 
+    # Similar to above, but don't use reference pixels.
+    mask = (input_model.dq>>31)&1 == 0
```

Comment:
I'm sure the math works out here, but bit shifting is hard to parse, and this isn't robust to potential changes in DQ flag values.  Can you please revise to use `dqflags.pixel['REFERENCE_PIXEL']`?